ci-operator/step-registry/openshift/e2e/openstack/ipi-e2e-only/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
You should replace this symlink with an actual OWNERS file with your approvers and your reviewers